### ISSUE TITLE
Solvent 'GBSA' did not validate

### ISSUE THREAD

**Issue by dprada** [choderalab/yank#957]
Sorry, me again :(
I already included in my yaml file the new blocks. :+1:
With explicit solvent works fine.

Now if I try to run a system with implicit solvent (including the examples) with:
```
solvents:
  GBSA:
    nonbonded_method: NoCutoff
    implicit_solvent: OBC2
```

I get:
```
2018-04-19 18:07:49,480: Setting CUDA platform to use precision model 'mixed'.
2018-04-19 18:07:49,585: ERROR - yank.experiment - Solvent 'GBSA' did not validate! Check the schema error below for details
ionic_strength: ['depends on these values: {''nonbonded_method'': [CutoffPeriodic,
    CutoffNonPeriodic, Ewald, PME]}']
solvent_model: [None not in the known solvent models map!, 'depends on these values:
    {''nonbonded_method'': [CutoffPeriodic, CutoffNonPeriodic, Ewald, PME]}']

Traceback (most recent call last):
  File "/opt/apps/conda/intel-2018.1.163_miniconda/envs/yank_0.21/bin/yank", line 11, in <module>
    load_entry_point('yank==0.21.1', 'console_scripts', 'yank')()
  File "/opt/apps/conda/intel-2018.1.163_miniconda/envs/yank_0.21/lib/python3.6/site-packages/yank/cli.py", line 73, in main
    dispatched = getattr(commands, command).dispatch(command_args)
  File "/opt/apps/conda/intel-2018.1.163_miniconda/envs/yank_0.21/lib/python3.6/site-packages/yank/commands/script.py", line 133, in dispatch
    yaml_builder = ExperimentBuilder(script=yaml_path, job_id=job_id, n_jobs=n_jobs)
  File "/opt/apps/conda/intel-2018.1.163_miniconda/envs/yank_0.21/lib/python3.6/site-packages/yank/experiment.py", line 636, in __init__
    self.parse(script)
  File "/opt/apps/conda/intel-2018.1.163_miniconda/envs/yank_0.21/lib/python3.6/site-packages/yank/experiment.py", line 748, in parse
    self._db.solvents = self._validate_solvents(yaml_content.get('solvents', {}))
  File "/opt/apps/conda/intel-2018.1.163_miniconda/envs/yank_0.21/lib/python3.6/site-packages/yank/experiment.py", line 1563, in _validate_solvents
    raise YamlParseError(error.format(solvent_id, yaml.dump(solvent_validator.errors)))
yank.experiment.YamlParseError: Solvent 'GBSA' did not validate! Check the schema error below for details
ionic_strength: ['depends on these values: {''nonbonded_method'': [CutoffPeriodic,
    CutoffNonPeriodic, Ewald, PME]}']
solvent_model: [None not in the known solvent models map!, 'depends on these values:
    {''nonbonded_method'': [CutoffPeriodic, CutoffNonPeriodic, Ewald, PME]}']
```

Sorry to bug you again. Thanks for your help.

**Comment by Lnaden** [comment]
There was a bug introduced in Cerberus 1.2 a few days ago which broke some logic in our schema. I fixed it in #952, but we have not done a bugfix yet. You can fix it yourself by doing

`conda install cerberus==1.1`.

I'll try to get bugfix out tomorrow for the weekend.

**Comment by dprada** [comment]
Right. Cerberus 1.1 fixes it.

Again, Thanks.
